app/__init__.py

```diff
@@ -12,3 +13,11 @@
 
 # Import all models here for easier importing.
 from app.users.models import User
+from app.users.resources import UserListResource, UserResource
+
+api = Api(app)
+
+
+api.add_resource(UserListResource, '/users')
```

Comment:
Perhaps add a comment right above stating that all resources that should be routed should be added here. Just for readability.